meteor/tsconfig-base.json

```diff
@@ -0,0 +1,28 @@
+{
+	"compilerOptions": {
+		"target": "es2019",
+		"experimentalDecorators": true,
+		"moduleResolution": "node",
+		"strictNullChecks": true,
+		"sourceMap": true,
+		"allowJs": false,
+		"jsx": "react",
+		"allowSyntheticDefaultImports": true,
+		"plugins": [
+			{
+				"name": "typescript-tslint-plugin"
+			}
+		],
+		"lib": ["dom", "es6", "dom.iterable", "scripthost", "es2017", "es2018"]
+	},
+	"include": [
+		"client/**/*",
+		"server/**/*",
+		"lib/**/*"
+	],
+	"exclude": [
+		"client/**/*",
```

Comment:
@nytamin this looks wrong..

Reply:
ah, yeah that's definitely wrong. I have probably pushed some temporary code..